include/caffe/solver.hpp

```diff
@@ -73,6 +73,7 @@ class Solver {
     return test_nets_;
   }
   int iter() { return iter_; }
+  void SetIter(int iter) { iter_ = iter; }
```

Comment:
Ah, I noticed this before but neglected to comment (sorry!): this should be `set_iter`; trivial functions use underscore_case (like the getter above).